src/ConnectedMode/SlCoreConnectionAdapter.cs

```diff
@@ -40,7 +40,9 @@ namespace SonarLint.VisualStudio.ConnectedMode;
 public interface ISlCoreConnectionAdapter
 {
     Task<AdapterResponse> ValidateConnectionAsync(ConnectionInfo connectionInfo, ICredentialsModel credentialsModel);
-    Task<AdapterResponseWithData<List<OrganizationDisplay>>> GetOrganizationsAsync(ICredentialsModel credentialsModel);
+
+    Task<AdapterResponseWithData<List<OrganizationDisplay>>> GetOrganizationsAsync(ICredentialsModel credentialsModel, CloudServerRegion cloudServerRegion);
```

Comment:
Why is this change in this PR, and not in SLVS-1842 Adapt ServerConnectionsRepositoryAdapter and SlCoreConnectionAdapter to support multi-region #6007 ?